src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+let summa = 0;
+
+const result = document.querySelectorAll('li');
```

Comment:
the same

Reply:
fix it, pls
https://github.com/mate-academy/js_get_data_DOM/pull/715/files#r1056909992